packages/docs/src/scripts/components/Footer.jsx

```diff
@@ -32,63 +32,102 @@ function renderLinks(links) {
 
   return urls.map(function(url) {
     return (
-      <li key={url}>
+      <dd key={url} className="ds-u-font-size--small ds-u-margin-left--0">
         <a href={url} target="_blank">
           {links[url]}
         </a>
-      </li>
+      </dd>
     );
   });
 }
 
 const Footer = () => {
   return (
-    <footer className="ds-u-fill--primary-alt-lightest ds-u-padding-y--6 ds-u-md-padding-bottom--3">
-      <div className="ds-u-padding-x--3 ds-u-margin-x--0">
+    <footer className="ds-u-fill--gray-lightest">
```

Comment:
Removing the padding means that the menu button covers up some text, maybe we should add the padding back for whatever breakpoint the menu button appears.
<img width="386" alt="Screen Shot 2019-10-16 at 4 54 54 PM" src="https://user-images.githubusercontent.com/5424713/66962096-ce80ef00-f035-11e9-9ef8-0491f084410f.png">

Its also worth noting the menu now has the same color as the footer and theres no contrast difference at all. I'm not sure if we should maybe change the menu button color? For reference heres how it looked originally
<img width="408" alt="Screen Shot 2019-10-16 at 4 54 46 PM" src="https://user-images.githubusercontent.com/5424713/66962194-038d4180-f036-11e9-9297-30d9b7e14029.png">
<img width="378" alt="Screen Shot 2019-10-16 at 4 56 35 PM" src="https://user-images.githubusercontent.com/5424713/66962200-06883200-f036-11e9-897c-fbb1c2b72995.png">

Reply:
I've adjusted the footer spacing and changed the menu button color